po/de.po

```diff
@@ -256,13 +256,13 @@ msgid "Build a subber"
 msgstr "Baut einen Kettentaucher"
 
 msgid "Build a target bot"
-msgstr ""
+msgstr "Baut einen Zielroboter"
```

Comment:
Minor inconsistency: TargetBot is translated as *Mobile Zielscheibe* (object name/SatCom).

Reply:
Adapted.